Update the sensor card expression for AMDC article

Daehoon-Sung opened the pull request:
This is to close #127.

I have updated:
- High-voltage sensor card
  - Corrected the wrong equation
  - Added to each of the AMDS card docs a section called ### Relationship Between Input and ADC voltage that provides an expression in the form of `INPUT = (VADC - OFFSET) x GAIN`
  - Included a different `Vadc` equation for each voltage card
- Low-voltage sensor card
  - Changed the original Vadc equation to the form of `INPUT = (VADC - OFFSET) x GAIN`
- Current sensor card
  - Changed the original Vadc equation to the form of `INPUT = (VADC - OFFSET) x GAIN`
  
# Notes

_Anything reviewers should be aware of when reviewing? Other related issues? Known problems? Future work?_

# Self-Review

_In this section, please self-review (answer all questions) on a suitable review checklist prior to requesting review from others. Select a review checklist based on what content is being merged in; see the [Review Checklists section](#review-checklists)._

# Reviewer Instructions

Reviewers, please copy and paste a suitable [review checklist](#review-checklists) into your review and answer all questions.

# Appendix

This section should be the same for all PRs. Do not edit this section when creating a PR.

## Review Checklists

Checklists maintained by the eLev lab for research repositories include:
- [Report checklist](https://github.com/Severson-Group/KnowledgeBase/blob/main/writing/write-repo-report.md#pr-review-checklist)
- [Publication archive checklist](https://github.com/Severson-Group/KnowledgeBase/blob/main/writing/archive-research-material.md) (select a checklist from the [Approach](https://github.com/Severson-Group/KnowledgeBase/blob/main/writing/archive-research-material.md#approach) that corresponds [to the step](https://github.com/Severson-Group/KnowledgeBase/blob/main/writing/archive-research-material.md#overview) this PR seeks to complete).
- [Standard checklist](https://github.com/Severson-Group/research-repo-template/blob/main/.github/pull_request_template.md) (if none of the above apply).

### Standard checklist

```markdown
1. Are all files under 300 kB (if not, please carefully assess whether it is worth committing them)? **Yes or No**
2. Are all files named according to the appropriate [naming convention](https://github.com/Severson-Group/research-repo-template?tab=readme-ov-file#file-naming), i.e., dash-case, camelCase, snake case? **Yes or No**
3. Do all Markdown files follow the [CONTRIBUTING article template](https://github.com/Severson-Group/.github/blob/main/CONTRIBUTING.md#markdown-documentation-template)? **Yes or No**
4. Do all links work in the material that the PR is adding? **Yes or No**
5. Is the PR configured to close the correct issue(s)? **Yes or No**
6. Did the PR fully address the `Approach` section of the issue(s) it is closing? **Yes or No**

Please work on addressing any **No** items.
```

Daehoon-Sung committed:
Update the sensor card equation

Daehoon-Sung committed:
Revise voltage equations and connector details

Updated voltage equations for Revision B and C, and added details about connectors.

Daehoon-Sung committed:
Update ADC voltage references in documentation

Daehoon-Sung committed:
Document ADC voltage relationship for sensor boards

Added section explaining the relationship between input and ADC voltage for low-voltage sensor boards.

Daehoon-Sung committed:
Change header from H4 to H3 for ADC voltage section

Daehoon-Sung committed:
Update index.md

Daehoon-Sung committed:
Update index.md

Daehoon-Sung committed:
Update voltage measurement equations for revisions

Daehoon-Sung committed:
Update voltage measurement formula for Revision B

Daehoon-Sung committed:
Update index.md

Daehoon-Sung commented:
Self-Review

1. Are all files under 300 kB (if not, please carefully assess whether it is worth committing them)? **Yes**
2. Are all files named according to the appropriate [naming convention](https://github.com/Severson-Group/research-repo-template?tab=readme-ov-file#file-naming), i.e., dash-case, camelCase, snake case? Not applicable to this PR
3. Do all Markdown files follow the [CONTRIBUTING article template](https://github.com/Severson-Group/.github/blob/main/CONTRIBUTING.md#markdown-documentation-template)? **Yes**
4. Do all links work in the material that the PR is adding? **Yes**
5. Is the PR configured to close the correct issue(s)? **Yes**
6. Did the PR fully address the `Approach` section of the issue(s) it is closing? **Yes**

Please work on addressing any **No** items.

noguchi-takahiro reviewed (dismissed):
Thank you @Daehoon-Sung. Looks good to me.

1. Are all files under 300 kB (if not, please carefully assess whether it is worth committing them)? **Yes**
2. Are all files named according to the appropriate [naming convention](https://github.com/Severson-Group/research-repo-template?tab=readme-ov-file#file-naming), i.e., dash-case, camelCase, snake case? **Not applicable**
3. Do all Markdown files follow the KB's [article template](https://github.com/Severson-Group/KnowledgeBase/blob/master/CONTRIBUTING.md#article-template)? **Yes**
4. Do all links work in the material that the PR is adding? **Yes**
5. Is the PR configured to close the correct issue(s)? **Yes**

@elsevers could your review this PR?

Daehoon-Sung committed:
Fix link reference in ADC voltage relationship section

Daehoon-Sung committed:
Fix link formatting in ADC voltage relationship section

Updated link formatting for consistency in the documentation.

Daehoon-Sung commented:
Hello Professor @elsevers . We have fixed the link issue. Could you review this? Thank you.

elsevers reviewed (changes requested):
Hello @Daehoon-Sung, thank you for good work here. 

## Current sensor card

I am wondering if it would be feasible to add a general expression for $I_{PRIMARY}$ in terms of $V_{REF}$, $R_{BURDEN}$, $R_a$, $R_b$, and $R_c$ right before the `Revision B` heading
 
<img width="700" alt="Image" src="https://github.com/user-attachments/assets/67d00840-07a6-4a33-9e7a-c91acac1d4f0" />

My thinking is that this would be helpful for students who modify these sensor cards for custom current ranges.

## High voltage card

I have a similar request to add a general expression for the input voltage:

<img width="600" alt="image" src="https://github.com/user-attachments/assets/1f8c2fa0-f22c-4590-880f-fe5c9ec83f40" />

I am thinking that we should rename $V_{MEAS}$ to be $V_{in}$, noting that this is how it is indicated in the block diagram at the top of the doc:

<img width="500" alt="image" src="https://github.com/user-attachments/assets/3d3945b7-f738-4515-91c8-42cd800c9a53" />

Daehoon-Sung committed:
Enhance index.md with ADC voltage equations

Added general expression for ADC voltage relationship and primary current calculation.

Daehoon-Sung committed:
Clarify ADC voltage relationship and revisions

Daehoon-Sung committed:
Fix equation formatting in index.md

Daehoon-Sung committed:
Update revision label from B to A, B

Daehoon-Sung committed:
Clarify ADC voltage relationship and revisions

Updated the relationship between measured voltage and ADC input voltage, including general expression and revisions A, B, and C.

Daehoon-Sung committed:
Revise sensor conversion ratio and resistor values

Updated sensor conversion ratio notation and clarified K_N value.

Daehoon-Sung committed:
Clarify sensor conversion ratio in documentation

Daehoon-Sung committed:
Update design parameters in index.md

Daehoon-Sung committed:
Fix redundancy in K_N notation in revisions A, B, C

Daehoon-Sung committed:
Fix voltage equation formatting in index.md

Daehoon-Sung committed:
Fix precision in voltage calculations in index.md

Daehoon-Sung committed:
Fix formatting in sensor cards index documentation

Daehoon-Sung committed:
Fix equation formatting in low-voltage index.md

Daehoon-Sung committed:
Fix reference voltage variable in equation

Corrected the reference voltage variable in the equation from 'V_{ref}' to 'V_{REF}' for consistency.

Daehoon-Sung committed:
Update index.md

Daehoon-Sung committed:
Fix reference to ADC version in calculations

Daehoon-Sung committed:
Fix formatting of PRIMARY current equation

noguchi-takahiro commented on source/accessories/amds/sensor-cards/high-voltage/index.md:
```diff
@@ -60,16 +68,32 @@ $$
+The different stages of the voltage sensor card described above convert the input voltage, to a voltage in the range of 0.2V - 4.5V. For the current design, 0V input voltage corresponds to 2.35V at the ADC input. The positive peak corresponds to 4.5V and the negative peak corresponds to 0.2V. 
+
+#### Relationship Between Input and ADC voltage
+From the equations provided in the [Level shift stage](#level-shift-stage) section, the general relationship between the measured voltage $V_{\rm MEAS}$ and the input voltage of ADC $V_{\text{ADC}}$ can be calculated, and the relationship for each revision of the current sensor board is provided below:
```
I wonder this `MEAS` should be updated to `IN`? Or we can keep this.

noguchi-takahiro reviewed (commented):
Thank you @Daehoon-Sung! I have only one comment that I wanted to double check. See below and let me know your thought. Here are rendered versions of these articles for you:

- [Current - AMDC Platform.pdf](https://github.com/user-attachments/files/23573070/Current.-.AMDC.Platform.pdf)
- [High Voltage - AMDC Platform.pdf](https://github.com/user-attachments/files/23573071/High.Voltage.-.AMDC.Platform.pdf)
- [Low Voltage - AMDC Platform.pdf](https://github.com/user-attachments/files/23573072/Low.Voltage.-.AMDC.Platform.pdf)

Daehoon-Sung committed:
Clarify ADC voltage relationship terminology

Updated the terminology from 'measured voltage' to 'input voltage' for clarity.

Daehoon-Sung committed:
Clarify input voltage terminology in ADC section

Updated the terminology for input voltage in the ADC relationship section.

Daehoon-Sung commented:
Hello, Professor @elsevers. We have implemented your feedback, and @noguchi-takahiro reviewed this. Could you review this PR? 

The main changes I have made are listed [here](https://github.com/Severson-Group/docs.amdc.dev/pull/144#issuecomment-3539450627). Thank you.

noguchi-takahiro reviewed (approved):
Thank you @Daehoon-Sung. Looks good to me.

1. Are all files under 300 kB (if not, please carefully assess whether it is worth committing them)? **Yes**
2. Are all files named according to the appropriate [naming convention](https://github.com/Severson-Group/research-repo-template?tab=readme-ov-file#file-naming), i.e., dash-case, camelCase, snake case? **Not applicable**
3. Do all Markdown files follow the KB's [article template](https://github.com/Severson-Group/KnowledgeBase/blob/master/CONTRIBUTING.md#article-template)? **Yes**
4. Do all links work in the material that the PR is adding? **Yes**
5. Is the PR configured to close the correct issue(s)? **Yes**

@elsevers could your review this PR?

Daehoon-Sung commented on source/accessories/amds/sensor-cards/current/index.md:
```diff
@@ -91,26 +91,31 @@ A single-ended ADC was selected. The ADC used is the Texas Instruments [ADS8860]
 
 $$
-V_{\text{out, RevC}} = 2.5126 + 0.034 I_{\text{PRIMARY}} \qquad { \rm [V]}
+I_{\text{PRIMARY}} = 29.4146 \times (V_{\text{ADC, RevC}} - 2.5126) \qquad \mathrm{[A]}
```
Hello Professor @elsevers 

Yes,  From [this comment](https://github.com/Severson-Group/docs.amdc.dev/pull/144#issuecomment-3539450627), 

## Previous
Previously, we used:

$$
V_{\text{out, RevB}} = 2.4922 + 0.034 I_{\text{p, RevB}}
$$

$$
V_{\text{out, RevC}} = 2.5126 + 0.034 I_{\text{p, RevC}}
$$

Then, we made them as a form of:

$$
I_{\text{p, RevB}} =\frac{1}{0.034} [V_{\text{out, RevB}} - 2.4922] = 29.4117647059 [V_{\text{out, RevB}} - 2.4922] 
$$

$$
I_{\text{p, RevC}} = \frac{1}{0.034} [V_{\text{out, RevB}} - 2.5126] = 29.4117647059 [V_{\text{out, RevB}} - 2.5126]
$$

which is the same value as in the AMDC Firmware. This is true, but to be more accurate, actually they are:

## Updated

$$
V_{\text{out, RevB}} = 2.4922 + 0.0341788 I_{\text{p, RevB}}
$$

$$
V_{\text{out, RevC}} = 2.5126 + 0.0339967 I_{\text{p, RevC}}
$$

Then, 

$$
I_{\text{PRIMARY, RevB}} = \frac{1}{0.0341788} \times (V_{\text{ADC, RevB}} - 2.4922)  = 29.2579 \times (V_{\text{ADC, RevB}} - 2.4922) 
$$

$$
I_{\text{PRIMARY, RevC}} = \frac{1}{0.0339967} \times (V_{\text{ADC, RevC}} - 2.5126)  = 29.4146 \times (V_{\text{ADC, RevC}} - 2.5126) 
$$

which is updated in this PR. 

Therefore, the reason that the gains are different is just due to the rounding error of the significant figures. The experiment was done using 0.034 gains. We validated this but did not validate it very accurately considering the significant figures. Since, the gains are almost same (0.034 \approx 0.0341788, 0.034 \approx 0.0339967), the experimental results will still be almost same. However, the updated version in this PR is more accurate one. 

I think we need to update the AMDC firmware either.

elsevers replied:
Thanks @Daehoon-Sung. I appreciate how thorough you are here. 

Can you get us set up with an issue in the amdc-firmware repo to update the gains? And then either you or @mohamed-dek1 can submit a PR to implement it (super simple, right?).

As soon as we have the issue up in amdc-firmware, I am ready to merge this.

Daehoon-Sung replied:
Thank you Professor @elsevers, @mohamed-dek1 !

elsevers reviewed (approved):
Thanks @Daehoon-Sung. Go ahead and `squash and merge`
